eng/Versions.props

```diff
@@ -36,7 +36,7 @@
     <SystemReflectionMetadataLoadContextVersion>6.0.0</SystemReflectionMetadataLoadContextVersion>
     <MicrosoftDeploymentDotNetReleasesVersion>1.0.0-preview1.1.21112.1</MicrosoftDeploymentDotNetReleasesVersion>
     <SystemManagementPackageVersion>4.6.0</SystemManagementPackageVersion>
-    <SystemCommandLineVersion>2.0.0-beta1.21601.2</SystemCommandLineVersion>
+    <SystemCommandLineVersion>2.0.0-beta1.21473.1</SystemCommandLineVersion>
```

Comment:
The command-line-api entries in Version.Details.xml need to be updated accordingly as well with this version.  Utilizing `darc update-dependencies` is a good way to ensure all entries are updated correctly.